data_selection/static/data_selection.js

```diff
@@ -0,0 +1,255 @@
+  "6531-61334-0009"
+];
+
+// Audio Instance
+var wavesurfer = WaveSurfer.create({
+  container: "#waveform",
+  waveColor: "#ADADAD",
+  scrollParent: true,
+  progressColor: "#0851DA",
+  barHeight: 2,
+  height: 41,
+  width: 100,
+  hideScrollbar: true,
+  audioRate: 1
+});
+
+// Load the audio sample
+var loadAudioSample = function(id) {
+  $("#stop-btn").addClass("display-none");
+  $("#play-btn").removeClass("display-none");
+
+  audioSample = audioSamples[id];
+  let audioPathFile = "data_selection/static/asset/" + audioSample;
+  wavesurfer.load(audioPathFile);
+  showCurrentRate();
+
+  let audioDuration = showAudioDuration();
+  document.getElementById("duration-time").innerHTML = audioDuration;
+
+  let currentTime = showCurrentTime();
+  document.getElementById("current-time").innerHTML = currentTime;
+};
+
+/* ---------- Audio options ---------- */
+
+// Play/Pause
+$(".btn-container").on("click", function() {
+  wavesurfer.playPause();
+
+  if ($("#play-btn").hasClass("display-none")) {
+    $("#play-btn").removeClass("display-none");
+  } else {
+    $("#play-btn").addClass("display-none");
+  }
+  if ($("#stop-btn").hasClass("display-none")) {
+    $("#stop-btn").removeClass("display-none");
+  } else {
+    $("#stop-btn").addClass("display-none");
+  }
+});
+
+//Audio current time
+wavesurfer.on("play", function() {
+  setInterval(function() {
+    var currentTime = showCurrentTime();
+    document.getElementById("current-time").innerHTML = currentTime;
+  }, 100);
+});
+
+// Reset audio at the start
+wavesurfer.on("finish", function() {
+  $("#stop-btn").addClass("display-none");
+  $("#play-btn").removeClass("display-none");
+  wavesurfer.stop();
+});
+
+// Set the audio speed rate
+$(".rate-option").on("click", function() {
+  let index = wavesurfer.getPlaybackRate();
+  const rates = [0.8, 1, 1.2];
+  if (this.id === "plus") {
+    wavesurfer.setPlaybackRate(rates[index + 1]);
+  } else if (this.id === "minus") {
+    wavesurfer.setPlaybackRate(rates[index - 1]);
+  }
+  let currentRate = wavesurfer.getPlaybackRate();
+  $("#audio-speed-rate").text(currentRate + "x");
+});
+
+// Navigation for the samples
+$(".sample-label span").on("click", function() {
+  let id = this.id;
+  loadAudioSample(this.id);
+  $(".sample-label span").removeClass("active");
+  $("#" + id).addClass("active");
+
+  // Hide the result panel when the user click on another audio sample
+  $("#results-section").addClass("display-none");
+});
+
+// Restart the audio sample (return to the start)
+$("#go-start").on("click", function() {
+  wavesurfer.stop();
+  $("#stop-btn").addClass("display-none");
+  $("#play-btn").removeClass("display-none");
+});
+
+// Finish the audio sample
+$("#go-end").on("click", function() {
+  wavesurfer.skipForward(10);
+  $("#stop-btn").addClass("display-none");
+  $("#play-btn").removeClass("display-none");
+});
+
+let input = "";
+
+$("#input-transcription")
+  .keypress(function(e) {
+    if (e.which === 13) {
```

Comment:
What does 13 means? You could define a const with an explicit name that would be equals to 13, this way we know